frontend/pages/index.js

```diff
@@ -217,3 +219,15 @@ export default function Home() {
     </div>
   )
 }
+
+
+Index.getInitialProps = async function() {
+  const res = await fetch('http://localhost:3000/api/counties/[id].js');
```

Comment:
Can you add in another constant for the frontend url and then use a string interpolation so we can change it later with some environment variables.

You should end up with something like this:  ``` `${FRONTEND_URL}/api/counties/[id].js` ```